themes/SuiteP/js/style.js

```diff
@@ -378,9 +378,10 @@ var isUserProfilePage = function() {
         module = $('#EditView_tabs').closest('form#EditView').find('input[name="module"]').val()
     }
     if(!module) {
-        module = module_sugar_grp1;
+        // ISSUE: breaks detailview and list views
```

Comment:
This line of code was break in the select records dropdown menu in the list and detail views. 

I was getting an error message stating that it was undefined. I have added the line back in. But i have a check to make sure that module_sugar_grp1 exists before assigning the value. Which prevents the issue while ensuring it can be assigned when needed.